### ISSUE TITLE
`shadcn-ui add button` generate invalid tailwind class when `cssVariable` is not set

### ISSUE THREAD

**Issue by SaiYs** [shadcn/ui#695]
When `cssVariable` is set as false in `components.json`, generated color classes with alpha value looks like "bg-neutral-900:90", which should be "bg-neutral-900/90".

Is seems that `applyColorMapping` in `transformer-css-value.ts` is just joining color and its alpha value with "-".

https://github.com/shadcn/ui/blob/7b5582e5d063c819a5cea7adf9edba159f918191/packages/cli/src/utils/transformers/transform-css-vars.ts#L162C1-L172C8

## Example

Full generated code is here.

button.tsx

```tsx
import * as React from "react"
import { Slot } from "@radix-ui/react-slot"
import { cva, type VariantProps } from "class-variance-authority"

import { cn } from "@/lib/cn"

const buttonVariants = cva(
  "inline-flex items-center justify-center rounded-md text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-neutral-400 disabled:pointer-events-none disabled:opacity-50 dark:focus-visible:ring-neutral-800",
  {
    variants: {
      variant: {
        default:
          "bg-neutral-900 text-neutral-50 shadow hover:bg-neutral-900:90 dark:bg-neutral-50 dark:text-neutral-900 dark:hover:bg-neutral-50:90",
        destructive:
          "bg-red-500 text-neutral-50 shadow-sm hover:bg-red-500:90 dark:bg-red-900 dark:text-red-50 dark:hover:bg-red-900:90",
        outline:
          "border border-neutral-200 bg-white shadow-sm hover:bg-neutral-100 hover:text-neutral-900 dark:border-neutral-800 dark:bg-neutral-950 dark:hover:bg-neutral-800 dark:hover:text-neutral-50",
        secondary:
          "bg-neutral-100 text-neutral-900 shadow-sm hover:bg-neutral-100:80 dark:bg-neutral-800 dark:text-neutral-50 dark:hover:bg-neutral-800:80",
        ghost: "hover:bg-neutral-100 hover:text-neutral-900 dark:hover:bg-neutral-800 dark:hover:text-neutral-50",
        link: "text-neutral-900 underline-offset-4 hover:underline dark:text-neutral-50",
      },
      size: {
        default: "h-9 px-4 py-2",
        sm: "h-8 rounded-md px-3 text-xs",
        lg: "h-10 rounded-md px-8",
        icon: "h-9 w-9",
      },
    },
    defaultVariants: {
      variant: "default",
      size: "default",
    },
  }
)

export interface ButtonProps
  extends React.ButtonHTMLAttributes<HTMLButtonElement>,
    VariantProps<typeof buttonVariants> {
  asChild?: boolean
}

const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
  ({ className, variant, size, asChild = false, ...props }, ref) => {
    const Comp = asChild ? Slot : "button"
    return (
      <Comp
        className={cn(buttonVariants({ variant, size, className }))}
        ref={ref}
        {...props}
      />
    )
  }
)
Button.displayName = "Button"

export { Button, buttonVariants }

```

components.json

```json
{
  "$schema": "https://ui.shadcn.com/schema.json",
  "style": "new-york",
  "rsc": true,
  "tailwind": {
    "config": "tailwind.config.js",
    "css": "src/styles/tailwind.css",
    "baseColor": "neutral",
    "cssVariables": false
  },
  "aliases": {
    "components": "@/components",
    "utils": "@/lib/cn"
  }
}

```

Thanks to your awesome project and I like a new update!

**Comment by dan5py** [comment]
Solved by #681

**Comment by SaiYs** [comment]
Sorry, I close this
